JustFakeIt/ExpectationExtensions.cs

```diff
@@ -53,7 +56,48 @@ public static bool MatchesActualHttpMethod(this HttpRequestExpectation expected,
 
         public static bool MatchesActualBody(this HttpRequestExpectation expected, string actualBody)
         {
-            return string.IsNullOrEmpty(expected.Body) || actualBody.Equals(expected.Body);
+            return MatchBody(expected.Body, actualBody);
```

Comment:
This changes the meaning of the test.  Instead of being an exact match, as previous, it's now a partial match.  This will give unexpected results.
This method should remain as an exact match for the entire body.